deployment/src/main/java/io/quarkiverse/openapi/generator/deployment/circuitbreaker/CircuitBreakerConfigurationParser.java

```diff
@@ -0,0 +1,55 @@
+package io.quarkiverse.openapi.generator.deployment.circuitbreaker;
+
+import java.util.List;
+import java.util.Map;
+import java.util.function.Function;
+import java.util.stream.Collectors;
+import java.util.stream.StreamSupport;
+
+import org.eclipse.microprofile.config.Config;
+
+public final class CircuitBreakerConfigurationParser {
```

Comment:
This is nice! But we need to have a holistic approach to reading these properties (see the authentication feature, we have a very similar way to get the properties). Not sure if we really need this since the `CodegenProvider` already has access to the properties in build time.

Reply:
As we discussed, this is a distinct case. So, we'll keep this way for now.